### PR TITLE
Updated index.md

### PR DESCRIPTION
**Pull request by dsmatlak**
* Recast "for creating" and replaced with "to create" in Intro section. 
* Added a bullet list for .NET Core projects in the .Net Core section.
* Fixed capitalization errors and removed extra spaces between words.

### CODE REVIEW

**Comment by PRMerger19** [comment]
@dsmatlak : Thanks for your contribution! The author, @asb3993, has been notified to review your proposed change.

File: mac/index.md [hunk 1]
```diff
@@ -41,7 +41,12 @@ F# is a strongly-typed functional programming language designed to run on .NET.
 
 ## .NET Core
 
-[.NET Core](https://www.microsoft.com/net/core#macos) is a platform for creating applications that run on Windows, Linux, and Mac. Visual Studio for Mac has support for loading, creating, running, and debugging .NET Core projects.
+[.NET Core](https://www.microsoft.com/net/core#macos) is a platform to create applications that run on Windows, Linux, and Mac. Visual Studio for Mac has support for these .NET Core projects: 
+
```
Comment (asb3993) [inline mac/index.md]: I don't think putting this as a bulleted list makes sense. These are all actions that are supported, not types of projects.

**Comment by asb3993** [comment]
Hey @dsmatlak, this looks great and I appreciate the additions. There's just one issue that I think needs to be fixed before this can be merged.

File: mac/index.md (hunk 1 above)
Comment (dsmatlak) [reply mac/index.md]: Thanks for the feedback. I will remove the bullet list.

**Comment by PRMerger14** [comment]
@dsmatlak : Thanks for your contribution! The author, @asb3993, has been notified to review your proposed change.

**Comment by asb3993** [comment]
Approved!

This will be merged shortly.

Thanks again.

**Pull request merged**